### PR TITLE
Add Furyagent role

### PR DESCRIPTION
**Pull request by nikever**
Move the furyagent role from [fury-kubernetes-aws](https://github.com/sighupio/fury-kubernetes-aws/tree/master/roles/furyagent) here.

### CODE REVIEW

**Comment by angelbarrera92** [comment]
LGTM!

File: roles/furyagent/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2021 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+---
+
+furyagent_uri: https://s3.wasabisys.com/sighup-releases/linux/furyagent
```
Comment (angelbarrera92) [inline roles/furyagent/defaults/main.yml:7]: I know you are just moving the role from one repo to another. But someone has to take care of updating the module.

As an example, we are not pushing the furyagent to wasabi anymore (as far as i know)

**Comment by nikever** [comment]
- I have merged this PR with the duplicated one (now closed).
- Added documentation for this module
- Remove this module from aws module https://github.com/sighupio/fury-kubernetes-aws/pull/49

**Review by angelbarrera92 — APPROVED** [review]
LGTM!

**Pull request merged**